server.js

```diff
@@ -104,6 +127,19 @@ function Weather(jsonObject){
   this.time = jsonObject.valid_date;
 }
 
+function Trail(jsonObject){
+  this.name = jsonObject.name;
+  this.location = jsonObject.location;
+  this.length = jsonObject.length;
+  this.stars = jsonObject.stars;
+  this.star_votes = jsonObject.starVotes;
+  this.summary = jsonObject.summary;
+  this.trail_url = jsonObject.url;
+  this.conditions = jsonObject.conditionDetails;
+  this.condition_date = jsonObject.conditionDate.split(' ',1);
+  this.condition_time = jsonObject.conditionDate.split(' ')[1];
```

Comment:
Utilizing the .split method to target the individual pieces of information in the object string is clever.